examples/Finding Pipelines with Available Updates.ipynb

```diff
@@ -0,0 +1,790 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f3f08d9",
+   "metadata": {},
+   "source": [
+    "# Find Pipelines with Available Updates\n",
+    "\n",
+    "This script will walk you through the API calls needed to determine which pipelines, configured in a given TDP Org, have a protocol with a newver artifact version available for use.\n",
+    "\n",
+    "This script uses the following python modules\n",
+    "\n",
+    "- `requests` - for handling TDP API calls\n",
+    "- `pandas` - for data processing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "501cdf55",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "import requests\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87e98663",
+   "metadata": {},
+   "source": [
+    "### Create the authorization object\n",
+    "\n",
+    "This `auth` object is used to handle authorization for requests.  It is required for (and common to) any API usage in TDP. Therefore it is commonly recommended to save this as a `.json` file for future use."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "ed3da3ae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auth = {\n",
+    "    \"api_url\": \"https://api.tetrascience.com/v1/\",\n",
+    "    \"auth_token\": \"\",\n",
+    "    \"org\": \"tetrascience\",\n",
+    "    \"verify_ssl\": True,\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "64ac559d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "headers = {\n",
+    "    \"ts-auth-token\": auth[\"auth_token\"],\n",
+    "    \"x-org-slug\": auth[\"org\"],\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "f1211897",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipelines_endpoint = \"pipeline/search\"\n",
+    "url = auth[\"api_url\"] + pipelines_endpoint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bc52905f",
+   "metadata": {},
+   "source": [
+    "#### Suppress Warnings\n",
+    "\n",
+    "If SSL verificaiton is contraindicated for the TDP environment, the below loop can get very noisy without suppressing warnings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "c8795027",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Suppresss HTTPS Verification warnings\n",
+    "requests.packages.urllib3.disable_warnings()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "493c0128",
+   "metadata": {},
+   "source": [
+    "### Get information for all piplines via Loop\n",
+    "\n",
+    "The piplines enpoint returns only 10 pipelines at a time.  We'll use the returned `\"hasNext\"` and `\"from\"` values to loop through the pages of results and capture the information on all the pipelines."
```

Comment:
endpoint.

also it can be up to 100. from API docs:
size
int32
Optional. Number of results to return starting from the page index given in the “from” parameter. Default is 10, Max is 100.